functional-tests.sh

```diff
@@ -226,8 +247,21 @@ function test_put_object()
 
     start_time=$(get_time)
     object_name="mc-test-object-$RANDOM"
-    assert_log_failure "$start_time" "${FUNCNAME[0]}" mc_cmd cp "${FILE_1_MB}" "${SERVER_ALIAS}/${BUCKET_NAME}/${object_name}"
-    assert_log_failure "$start_time" "${FUNCNAME[0]}" mc_cmd rm "${SERVER_ALIAS}/${BUCKET_NAME}/${object_name}"
+    assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd cp "${FILE_1_MB}" "${SERVER_ALIAS}/${BUCKET_NAME}/${object_name}"
+    assert_success "$start_time" "${FUNCNAME[0]}" mc_cmd rm "${SERVER_ALIAS}/${BUCKET_NAME}/${object_name}"
+
+    log_success "$start_time" "${FUNCNAME[0]}"
+}
+
+function test_put_object_error()
+{
+
+    show "${FUNCNAME[0]}"
+    start_time=$(get_time)
+
+    long_name=$(printf '=%.0s' {1..1100})
+    object_name="mc-test-object-$long_name"
```

Comment:
I think `object_name=$(printf "mc-test-object-%01100d" 1)` would be simpler and understandable.